src/server/pps/server/api_server.go

```diff
@@ -1568,6 +1572,107 @@ func (a *apiServer) getLogsFromStats(pachClient *client.APIClient, request *pps.
 	return eg.Wait()
 }
 
+// GetLogs implements the protobuf pps.GetLogs RPC
+func (a *apiServer) GetLogsLoki(request *pps.GetLogsRequest, apiGetLogsServer pps.API_GetLogsServer) (retErr error) {
```

Comment:
nit: similar to e.g. `getLogsFromStats`, should this function be lowercased since it's not part of the gRPC public API?

Reply:
Yeah that seems reasonable.